File: HRM_Client_Mobile/HRM_Client/lib/models/JwtToken.dart
class JwtToken{
  int id;
  String tokenType;
  String accessToken;
  DateTime expiredTime;

  JwtToken({this.id, this.tokenType, this.accessToken, this.expiredTime});

  factory JwtToken.fromJson(Map<String, dynamic> json) {
    return JwtToken(
        id: json["id"],
        tokenType: json["tokenType"],
        accessToken: json["accessToken"],
        expiredTime: DateTime.parse(json["expiredTime"])
    );
  }
}

File: HRM_BE/HRM.Core/Models/Users/JwtToken.cs
using System;

namespace HRM.Core.Models.Users
{
    public class JwtToken
    {
        public string Id { get; set; }
        public string TokenType { get; set; } = "JWT Bearer";
        public string AccessToken { get; set; }
        public DateTime ExpiredTime { get; set; }
    }
}
